pages/_app.js

```diff
@@ -22,7 +22,7 @@ function MyApp({ Component, pageProps, router }) {
       <SEO
         title="Web APIs Playground - Create, Share, Learn JavaScript Web APIs"
         description="The Web APIs Playground is a project to showcase the JavaScript Web APIs with examples and demonstrations. Client-side JavaScript APIs provides wrapper functions for many low-level tasks."
-        icon="/favicon.ico"
+        icon="../public/assets/Logo.png"
```

Comment:
@kvnal Can we create a favicon.ico from the logo and set it? Btw I do not see the favicon appearing right on the preview app. Could you plz check?

https://webapis-playground-ljzr5xrv2-thoughtdump.vercel.app/

Reply:
@kvnal just use a online png to favicon converter

Reply:
Fixed pls review!

Reply:
I feel like the logo is very tiny. Is the logo full width without any blank spaces around? Is it crop properly just to contain the logo and no blank spaces around?

Reply:
I agree